### PR TITLE
Cofense Triage Release 2.1.13

### PR DESCRIPTION
**Pull request by crestdatasystems**
## Contributing to Cortex XSOAR Content
Make sure to register your contribution by filling the [contribution registration form](https://forms.gle/XDfxU4E61ZwEESSMA)

**The Pull Request will be reviewed only after the contribution registration form is filled.**

## Status
- [ ] In Progress
- [x] Ready
- [ ] In Hold - (Reason for hold)

## Related Issues
fixes: link to the issue

## Description
Added the new `cofense-report-attachment-payload-list` command.

## Screenshots
Paste here any images that will help the reviewer

## Minimum version of Cortex XSOAR
- [ ] 6.0.0
- [ ] 6.1.0
- [ ] 6.2.0
- [x] 6.5.0

## Does it break backward compatibility?
   - [ ] Yes
       - Further details:
   - [x] No

## Must have
- [x] Tests
- [x] Documentation

### CODE REVIEW

File: Packs/CofenseTriage/Integrations/CofenseTriagev3/README.md [hunk 1]
```diff
@@ -715,6 +715,115 @@ There is no context output for this command.
...
+| Cofense.AttachmentPayload.relationships.reports.links.self | String | Link to retrieve the report with attachments containing the payload. | 
+| Cofense.AttachmentPayload.relationships.reports.links.related | String | Link to retrieve the detailed information of the report with attachments containing the payload. | 
+
+
+#### Command Example
+```!cofense-report-attachment-payload-list id=47024 page_size=2```
+
+#### Context Example
+```json
+{
```
[the remark this answers is not in the document]
Comment (crestdatasystems) [reply Packs/CofenseTriage/Integrations/CofenseTriagev3/README.md:767]: @michal-dagan, thanks for pointing this out. I've made the suggested changes.

**Comment by crestdatasystems** [comment]
> Hi @crestdatasystems, Thank you for your contribution! Great work :) Let me know whether you want to record and share with me a small demo video, or if you want to schedule a meeting for a live demo, whichever works best for you. thanks again

Hi @michal-dagan, Thanks for the quick review, I've shared a recording with you over Slack. Please let me know if you need anything else from our side.

**Comment by michal-dagan** [comment]
> > Hi @crestdatasystems, Thank you for your contribution! Great work :) Let me know whether you want to record and share with me a small demo video, or if you want to schedule a meeting for a live demo, whichever works best for you. thanks again
> 
> Hi @michal-dagan, Thanks for the quick review, I've shared a recording with you over Slack. Please let me know if you need anything else from our side.

Great Demo, thanks!
'Run Validations' failed on the docker images: 
please update:
`demisto-sdk format -ud -i Packs/CofenseTriage/Integrations/CofenseTriagev3/CofenseTriagev3.yml`
`demisto-sdk format -ud -i Packs/CofenseTriage/Scripts/CofenseTriageReportDownload/CofenseTriageReportDownload.yml`
`demisto-sdk format -ud -i Packs/CofenseTriage/Scripts/CofenseTriageThreatEnrichment/CofenseTriageThreatEnrichment.yml`

**Comment by michal-dagan** [comment]
> 
> Great Demo, thanks! 'Run Validations' failed on the docker images: please update: `demisto-sdk format -ud -i Packs/CofenseTriage/Integrations/CofenseTriagev3/CofenseTriagev3.yml` `demisto-sdk format -ud -i Packs/CofenseTriage/Scripts/CofenseTriageReportDownload/CofenseTriageReportDownload.yml` `demisto-sdk format -ud -i Packs/CofenseTriage/Scripts/CofenseTriageThreatEnrichment/CofenseTriageThreatEnrichment.yml`

Please update also the release-note accordingly:

```
Validating Packs/CofenseTriage/ReleaseNotes/2_1_13.md as releasenotes
[ERROR][INFO] : Packs/CofenseTriage/ReleaseNotes/2_1_13.md: [RN111] - The 2_1_13.md release notes file contains incompatible Docker images:
- Cofense Triage v3: Release notes file has dockerimage: demisto/python3:3.10.11.61265 but the YML file has dockerimage: demisto/python3:3.10.12.62631
- CofenseTriageThreatEnrichment: Release notes file has dockerimage: demisto/python3:3.10.11.61265 but the YML file has dockerimage: demisto/python3:3.10.12.62631
- CofenseTriageReportDownload: Release notes file has dockerimage: demisto/python3:3.10.11.61265 but the YML file has dockerimage: demisto/python3:3.10.12.62631
```

To fix this please run: 'demisto-sdk update-release-notes -i Packs/CofenseTriage'

**Comment by crestdatasystems** [comment]
> > Great Demo, thanks! 'Run Validations' failed on the docker images: please update: `demisto-sdk format -ud -i Packs/CofenseTriage/Integrations/CofenseTriagev3/CofenseTriagev3.yml` `demisto-sdk format -ud -i Packs/CofenseTriage/Scripts/CofenseTriageReportDownload/CofenseTriageReportDownload.yml` `demisto-sdk format -ud -i Packs/CofenseTriage/Scripts/CofenseTriageThreatEnrichment/CofenseTriageThreatEnrichment.yml`
> 
> Please update also the release-note accordingly:
> 
> ```
> Validating Packs/CofenseTriage/ReleaseNotes/2_1_13.md as releasenotes
> [ERROR][INFO] : Packs/CofenseTriage/ReleaseNotes/2_1_13.md: [RN111] - The 2_1_13.md release notes file contains incompatible Docker images:
> - Cofense Triage v3: Release notes file has dockerimage: demisto/python3:3.10.11.61265 but the YML file has dockerimage: demisto/python3:3.10.12.62631
> - CofenseTriageThreatEnrichment: Release notes file has dockerimage: demisto/python3:3.10.11.61265 but the YML file has dockerimage: demisto/python3:3.10.12.62631
> - CofenseTriageReportDownload: Release notes file has dockerimage: demisto/python3:3.10.11.61265 but the YML file has dockerimage: demisto/python3:3.10.12.62631
> ```
> 
> To fix this please run: 'demisto-sdk update-release-notes -i Packs/CofenseTriage'

@michal-dagan, Made the suggested changes.

**Review by michal-dagan — APPROVED** [review]

**Pull request merged**